app/src/main/java/org/mozilla/focus/settings/ManualAddSearchEngineSettingsFragment.java

```diff
@@ -71,6 +72,12 @@ public boolean onOptionsItemSelected(MenuItem item) {
         }
     }
 
+    @Override
+    public void onPause() {
+        super.onPause();
+        ViewUtils.hideKeyboard(getActivity().getCurrentFocus());
```

Comment:
Is there any chance getCurrentFocus() could return null here?

Reply:
From the testing I've done getCurrentFocus() never returned null. The tests I tried was:

* Rotating the screen whilest ManualAddSearchEngineSettingsFragment was being displayed
* Closing the app and reopening after some time
* Closing the app, rotating the screen then returning to the app
* Clicking "cancel" when the fragment was being displayed
* Clicking "save" when the fragment was being displayed

To be safe I'll wrap it within an if-statement ensuring the view returned is not null

Reply:
Great. That's definitely more than I tested.. :)

Reply:
Now that I think of it let's change the code in ViewUtils.hideKeyboard() to check that the View passed to it is not null, thus allowing us to call hideKeyboard() without having to check if the view passed is null every time

Reply:
Yep, sounds good!